src/scheduler.cpp

```diff
@@ -87,19 +87,25 @@ namespace marl {
 ////////////////////////////////////////////////////////////////////////////////
 thread_local Scheduler* Scheduler::bound = nullptr;
 
+CLANG_NO_SANITIZE_MEMORY
 Scheduler* Scheduler::get() {
   return bound;
 }
 
+CLANG_NO_SANITIZE_MEMORY
+void Scheduler::setBound(Scheduler* scheduler) {
```

Comment:
nit: I feel this should probably be called `set()` for symmetry with `get()`.

Reply:
Yeah, sorry about the lack of symmetry.
I try to avoid naming functions after commonly used `std` types (like `vector`, `set`, `map`, etc).

Reply:
Feel free to change it (or ask me to change it) if you prefer the function being called `set`